### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by bosyk**
- documentation
- legacy pylana support
- packaging

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,55 @@
...
+```python
+response_list = api.get('/api/v2/dashboards')
 ```
 
+will return a response with a list of dashboard metadata.
+
 ## License
 [Apache License 2.0](http://www.apache.org/licenses/)
+
+## PyPI
```
Comment (Meissnerinc) [inline README.md]: Should we add the directions from the Contributing markdown here?

File: tests/test_user_management.py [hunk 2]
```diff
@@ -14,7 +14,10 @@ def setUpClass(cls) -> None:
...
 
     def test_success(self):
-        actual = get_user_information(**self.credentials)
+        url = self.credentials['scheme'] + '://' \
+            + self.credentials['host'] + ':' \
+            + str(self.credentials['port'])
+        actual = get_user_information(url, self.credentials['token'])
 
         # TODO: reduced to required
         expected_keys = [
```
Comment (Meissnerinc) [inline tests/test_user_management.py:23]: The expected keys are all correct, perhaps we can get rid of this TODO?

Comment (Meissnerinc) [inline tests/test_user_management.py:22]: The expected keys are all correct, perhaps we can get rid of this TODO?

**Review by Meissnerinc — CHANGES_REQUESTED** [review]

**Review by Meissnerinc — COMMENTED** [review]
Added two comments, only one requires (re-)action

File: README.md (hunk 1 above)
Comment (bosyk) [reply README.md]: Oh, this is a residue. We can put directions here. The `CONTRIBUTING.md` is a kind-of standard, but it doesn't hurt to refer.

File: tests/test_user_management.py (hunk 2 above)
Comment (bosyk) [reply tests/test_user_management.py:22]: done

**Review by Meissnerinc — APPROVED** [review]
good to go

**Pull request merged**